bip-emilengler-redefinition-of-the-term-address.mediawiki

```diff
@@ -0,0 +1,54 @@
+<pre>
+  BIP: UNDEFINED
+  Titel: Redefinition of the term address
+  Author: Emil Engler <me@emilengler.com>
```

Comment:
I'd suggest to find one co-author, so that potential changes to the BIP can be addressed by either

Reply:
Good idea, I will ask on the ML

Reply:
If no one else steps up, you can put me in.